Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Update the NODE_VERSION arg in docker-compose.yml to pick a Node version: 18, 16, 14
+ARG NODE_VERSION=18
+
+# Default ARGs are set in this file. To update them, update them in the devcontainer.json file
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:${NODE_VERSION}
+
+# VARIANT can be either 'hugo' for the standard version or 'hugo_extended' for the extended version.
+ARG VARIANT=hugo_extended
+
+# VERSION can be either 'latest' or a specific version number
+ARG VERSION=latest
+
+RUN mkdir /code
+
+WORKDIR /code
+
+# Download Hugo
+RUN apt-get update && apt-get install -y ca-certificates openssl git curl golang && \
+    rm -rf /var/lib/apt/lists/* && \
+    case ${VERSION} in \
+    latest) \
+    export VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4)}') ;;\
+    esac && \
+    echo ${VERSION} && \
+    case $(uname -m) in \
+    aarch64) \
+    export ARCH=ARM64 ;; \
+    *) \
+    export ARCH=64bit ;; \
+    esac && \
+    echo ${ARCH} && \
+    wget -O ${VERSION}.tar.gz https://github.com/gohugoio/hugo/releases/download/v${VERSION}/${VARIANT}_${VERSION}_Linux-${ARCH}.tar.gz && \
+    tar xf ${VERSION}.tar.gz && \
+    mv hugo /usr/bin/hugo
+
+# Hugo dev server port
+EXPOSE 1313
+CMD ["hugo" , "server", "-D", "--bind", "0.0.0.0" ]
```

Comment:
Can you explain why you included this line? I'd like to keep it as minimal as possible.

Reply:
I was facing some errors so tried binding
Also, I understand my previous approach was not correct
Hope this commit is correct